docs/glossary.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Glossary
+sidebar_label: Glossary
+keywords:
+- Harvester
+- glossary
+- terminology
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.3/glossary"/>
+</head>
+
+**guest cluster** / **guest Kubernetes cluster**
+
+Group of integrated Kubernetes worker machines that run in VMs on top of a Harvester cluster. 
+
+You can create RKE1, RKE2, and K3s guest clusters using the Harvester and Rancher interfaces. Creating guest clusters involves pulling images from either the internet or a private registry.
+
+Guest clusters form the main infrastructure for running container workloads. Certain versions of Harvester and Rancher allow you to deploy container workloads [directly to Harvester clusters](./rancher/rancher-integration/#harvester-baremetal-container-workload-support-experimental) (with some limitations).
+
+**guest node** / **guest cluster node**
+
+Kubernetes worker VM that uses guest cluster resources to run container workloads. 
+
+Guest nodes are managed through a control plane that controls pod-related activity and maintains the desired cluster state.
+
+**Harvester cluster** 
+
+Group of integrated physical servers (hosts) on which the Harvester hypervisor is installed. These servers pool compute and storage resources to provide an environment for running VMs that can host guest clusters.
```

Comment:
I'm not sure `pool` is the right word here - especially for storage.
While the strict dictionary definition of a pool doesn't specify anything that Harvester doesn't support, storage-pools are usually referring to _aggregations_ of multiple devices in such a way, that the volumes' storage capacities can exceed the capacity of any single device. This specifically is not the case for Longhorn, which provides the (default) storage engine for Harvester workloads. With Longhorn the storage is replicated across devices, but the capacity of the devices isn't pooled together. As a result, the maximum volume size can not exceed the capacity of the individual devices.
While it's theoretically possible to aggregate multiple disks within one node by using Longhorn on top of an LVM, this configuration is AFAIK not supported officially and I'm sure it would be quite a challenge to even set it up with Harvester.
Similarly, the maximum compute a single workload (i.e. VM) on Harvester can have is the compute a single node can provide, so there isn't really any pooling going on.

How about:

> These servers manage compute, memory and storage resources together to provide an environment for running VMs.

This should accurately capture that Harvester orchestrates these resources across the entire cluster, but this doesn't imply features that Harvester doesn't support.

Reply:
I understand your concern about using `pool` given how Longhorn and single-node clusters function. We can use the wording you suggested (with one small change):

> These servers collectively manage compute, memory, and storage resources to provide an environment for running VMs.